src/SFA.DAS.Provider.PR.Web/Models/EmployerDetailsViewModel.cs

```diff
@@ -34,6 +35,8 @@ public class EmployerDetailsViewModel
 
     public string LastActionText { get; set; } = null!;
 
+    public bool ShowLastActionText => !string.IsNullOrEmpty(LastActionText);
```

Comment:
I would update this to IsNullOrWhiteSpace

Reply:
ok